src/Morphir/Visual/Components/Theme.elm

```diff
@@ -0,0 +1,24 @@
+module Morphir.Visual.Components.Theme exposing (..)
+
+import Element
+
+
+type alias Theme =
+    { fontSize : Int
+    , smallSpacing : Int
+    , mediumSpacing : Int
+    , largeSpacing : Int
+    , smallPadding : Int
+    , mediumPadding : Int
+    , largePadding : Int
+    }
+
+
+type alias Style =
+    { fontSize : Int
+    }
+
```

Comment:
Maybe you should rename this to `Config` and create a function to derive the `Theme` from the `Config`. Something like this:

```elm
fromConfig : Config -> Theme
fromConfig config =
    { fontSize = config.fontSize |> Maybe.withDefault 12 }
```

Assuming that you change fontSize to be optional in the config which make it easier to use I think.